src/utils/sanitize.test.ts

```diff
@@ -1,6 +1,21 @@
 import { sanitizeArray, sanitizeObject, sanitizeString } from "./sanitize";
 
+var mockedLibrary: any;
```

Comment:
`var`?!?!?!

Reply:
`var`! On purpose! Jest has some kind of [weird hoisting](https://github.com/jestjs/jest/issues/11153#issuecomment-803639307) (which, TBH, I don't really understand), so using `let` gave me a `ReferenceError: Cannot access 'mockedLibrary' before initialization`.

But apparently `var`'s hoisting is even hoistier than jest's, so that `mockedLibrary` can be in scope before the jest initialization code runs.

It feels pretty weird though. Would love to hear if there's a better way to do what I need here 👀👂